source/Fleet.cpp

```diff
@@ -252,19 +251,11 @@ bool Fleet::IsValid(bool requireGovernment) const
 
 void Fleet::RemoveInvalidVariants()
 {
-	auto IsInvalidVariant = [](const Variant &v) noexcept -> bool
-	{
-		return !v.IsValid();
-	};
-	auto firstInvalid = find_if(variants.begin(), variants.end(), IsInvalidVariant);
-	if(firstInvalid == variants.end())
-		return;
-
 	// Ensure the class invariant can be maintained.
```

Comment:
This comment isn't accurate since we no longer have to manually update `total`. We only get the value now for logging purposes.